meta-facebook/yv35-cl/src/ipmi/plat_ipmi.c

```diff
@@ -909,6 +910,110 @@ void pal_OEM_GET_FW_VERSION(ipmi_msg *msg) {
   return;
 }
 
+void pal_OEM_SET_JTAG_TAP_STA(ipmi_msg *msg) {
```

Comment:
Can we assume that msg will never be NULL for any reason?

Might be best to check for null before using it.